Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: deploy-stack remove-stack build-nginx
+
+STACK_NAME ?= cfa_opencti
+COMPOSE_FILE?= docker-compose.yml
+NGINX_TAG?=0.0.1
```

Comment:
Lets track actual Nginx version. So this would be something like `1.25.4-1`, `1.25.4-1`, etc. while we're using `1.25.4`, then again `1.25.5-1`, `1.25.4-1` when we upgrade to `1.25.5`, etc.